.github/workflows/CI-push-main.yml

```diff
@@ -0,0 +1,21 @@
+name: CI - Push Main
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  swiftlint:
+    name: SwiftLint
+    uses: ./.github/workflows/lint.yml
+    secrets: inherit
+  test:
+    name: Testing CodeEdit
+    needs: swiftlint
+    uses: ./.github/workflows/tests.yml
+    secrets: inherit
+  deploy:
+    name: Deploying CodeEdit [Prerelease]
```

Comment:
Hi Luke, 

i suggest a small change
```suggestion
    if: github.repository_owner == 'CodeEdit'
    name: Deploying CodeEdit [Prerelease]
```

So that the building is only triggered on the CodeEdit repo

Reply:
Good idea! But in this case `repository_owner` should be `CodeEditApp` according to the GitHub docs.

Reply:
Oops my fault, i picked the repo name instead of the org name. (fixed)